CRM/Campaign/Task.php

```diff
@@ -143,8 +120,8 @@ public static function &permissionedTaskTitles($permission) {
   public static function getTask($value) {
     self::tasks();
     if (!$value || !CRM_Utils_Array::value($value, self::$_tasks)) {
-      // make the interview task by default
-      $value = 1;
+      // make the print task by default
```

Comment:
@mattwire this comment is incorrect - otherwise changes in this commit look good. 

Note the comment wouldn't be needed if the function were called 'getDefaultTask

Reply:
Well spotted :-) The default task stuff is all a bit wierd... but given the scale of the code changes I'm making here I didn't want to make any functional changes.  There is a PR raised by @johanv (https://github.com/civicrm/civicrm-core/pull/11596) which is waiting for all this to be merged that touches on the default task